server/storage/mvcc/kv_test.go

```diff
@@ -225,34 +226,78 @@ func testKVRangeLimit(t *testing.T, f rangeFunc) {
 		limit   int64
 		wcounts int64
 		wkvs    []mvccpb.KeyValue
+		wmore   bool
 	}{
 		// no limit
-		{-1, 3, kvs},
+		{-1, 3, kvs, false},
 		// no limit
-		{0, 3, kvs},
-		{1, 3, kvs[:1]},
-		{2, 3, kvs[:2]},
-		{3, 3, kvs},
-		{100, 3, kvs},
+		{0, 3, kvs, false},
+		{1, 3, kvs[:1], true},
+		{2, 3, kvs[:2], true},
+		{3, 3, kvs, false},
+		{100, 3, kvs, false},
 	}
 	for i, tt := range tests {
-		r, err := f(s, []byte("foo"), []byte("foo3"), RangeOptions{Limit: tt.limit})
-		if err != nil {
-			t.Fatalf("#%d: range error (%v)", i, err)
-		}
-		if !reflect.DeepEqual(r.KVs, tt.wkvs) {
-			t.Errorf("#%d: kvs = %+v, want %+v", i, r.KVs, tt.wkvs)
-		}
-		if r.Rev != wrev {
-			t.Errorf("#%d: rev = %d, want %d", i, r.Rev, wrev)
-		}
-		if tt.limit <= 0 || int(tt.limit) > len(kvs) {
-			if r.Count != len(kvs) {
-				t.Errorf("#%d: count = %d, want %d", i, r.Count, len(kvs))
+		tt := tt
+		t.Run(fmt.Sprint(i), func(t *testing.T) {
+			r, err := f(s, []byte("foo"), []byte("foo3"), RangeOptions{Limit: tt.limit})
+			require.Nil(t, err)
+			require.Equal(t, tt.wkvs, r.KVs)
+			require.Equal(t, wrev, r.Rev)
+			if tt.limit <= 0 || int(tt.limit) > len(kvs) {
+				require.Equal(t, len(kvs), r.Count)
+			} else {
+				require.Equal(t, int(tt.wcounts), r.Count)
 			}
-		} else if r.Count != int(tt.wcounts) {
-			t.Errorf("#%d: count = %d, want %d", i, r.Count, tt.limit)
+			require.Equal(t, tt.wmore, r.More)
+		})
+	}
+}
+
+func TestKVRangeMaxBytes(t *testing.T)    { testKVRangeMaxBytes(t, normalRangeFunc) }
+func TestKVTxnRangeMaxBytes(t *testing.T) { testKVRangeMaxBytes(t, txnRangeFunc) }
+
+func testKVRangeMaxBytes(t *testing.T, f rangeFunc) {
+	b, tmpPath := betesting.NewDefaultTmpBackend(t)
+	s := NewStore(zaptest.NewLogger(t), b, &lease.FakeLessor{}, StoreConfig{})
+	defer cleanup(s, b, tmpPath)
```

Comment:
remove `tmpPath`?

Reply:
Thanks! This `cleanup` was refactored after my first push.